charts/tekton-apps/templates/serviceaccounts.yaml

```diff
@@ -11,8 +11,15 @@ metadata:
 
 {{- if $project.enabled }}
 {{- range $component := $project.components }}
+{{- /* initialize `argocd` variable with either `$component.argocd` dict if it exists or with empty dict */}}
+{{ $argocd := ternary $component.argocd dict (hasKey $component "argocd") -}}
+
+{{- /* initialize `wordpress` variable with either `true` if it exists or with `false` */}}
+{{ $wordpress := ternary true false (hasKey $argocd "wordpress") -}}
+
 {{- $data := dict "project"           $project
                   "component"         $component
+                  "argocd"            $argocd
```

Comment:
Let's stick with original structure of passed args.

Reply:
done